src/main/java/core/basesyntax/geometry/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.geometry.figures;
+
+import core.basesyntax.geometry.Figure;
+
+public class Circle extends Figure {
+    private static final double NUMBER_P = 3.14;
```

Comment:
it's a redundant constant, take look at https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html